components/builder-web/app/origins-page/OriginsPageComponent.ts

```diff
@@ -17,62 +17,84 @@ import {acceptOriginInvitation, fetchMyOriginInvitations, fetchMyOrigins}
+                        </table>
+                    </div>
+                    <div *ngIf="invitations.size > 0">
+                        <h3>Invitations</h3>
+                        <ul>
+                            <li *ngFor="let invitation of invitations" class="hab-item-list hab-no-select">
+                            <h3 class="hab-item-list--title">{{invitation.origin_name}}</h3>
+                            <button
+                                class="count"
+                                (click)="acceptInvitation(invitation.origin_invitation_id, invitation.origin_name)">
+                                Accept Invitation
+                                </button>
+                            </li>
+                        </ul>
+                    </div>
                 </div>
             </div>
-            <div *ngIf="origins.size > 0">
-                <ul class="hab-origins-list">
-                    <li *ngFor="let origin of origins">
-                        <a [routerLink]="['/origins', origin.name]"
-                           class="hab-item-list">
-                            <div class="hab-item-list--title">
-                                <h3>{{origin.name}}</h3>
-                            </div>
-                        </a>
-                    </li>
-                </ul>
+            <div class="page-body--sidebar">
+                <p><b class="uppercase">ORIGINS</b><p>
+                <a class="button create" [routerLink]="['/origins', 'create']">Create New Origin</a>
+                <p>Origins are used to manage ownership over a set of packages.
+                More specifically, each origin will have a unique key pair that is used by members
+                to upload and download packages</p>
+                <p>While origins themselves are always publically visible in the depot,
+                their package builds may be either public or private</p>
+                <p>When (or after) creating an origin, you can set the default privacy for any package
+                build uploaded to the depot</p>
             </div>
-            <div *ngIf="invitations.size > 0">
-                <h3>Invitations</h3>
-                <ul>
-                    <li *ngFor="let invitation of invitations" class="hab-item-list hab-no-select">
-                       <h3 class="hab-item-list--title">{{invitation.origin_name}}</h3>
-                       <button
-                           class="count"
-                           (click)="acceptInvitation(invitation.origin_invitation_id, invitation.origin_name)">
-                           Accept Invitation
-                        </button>
-                    </li>
-                </ul>
-            </div>
-        </div>
+      </div>
     </div>`,
 })
 
 export class OriginsPageComponent implements OnInit {
-    constructor(private store: AppStore) { }
+    constructor(private store: AppStore, private router: Router) { }
```

Comment:
Technically this is using a "private" member, but the linter isn't picking it up because it's not in a template string.   Probably better to either delegate to a public getter or call a public `navigate` handler that calls the pribate `router` instance.